Source/Core/Common/Arm64Emitter.h

```diff
@@ -350,6 +350,12 @@ enum class RoundingMode
   Z,  // round towards zero
 };
 
+enum class GPRSize
+{
+  _32,
+  _64,
```

Comment:
I can't find anything about a leading underscore followed by a digit being reserved, but this still bothers me. Would you mind changing this to `B32` or something like that instead?

Reply:
Yeah, it bothers me a little too, I just wasn't sure what to use instead. B sounds good enough, so I'll go with that. There is the problem that "8B" in AArch64 assembly syntax means 8 *bytes*, but since you can't fit 32 bytes in a GPR I don't think there's any real potential for confusion.